chart/kubeapps/values.schema.json

```diff
@@ -1,135 +1,147 @@
 {
   "$schema": "http://json-schema.org/schema#",
-  "type": "object",
   "properties": {
-    "frontend": {
-      "type": "object",
```

Comment:
I have tried this and while `type: object` is not necessary from a JSONSchema point of view, we are using it to render subsections. Without it the form is not properly rendered:

![Screenshot from 2021-03-24 16-39-01](https://user-images.githubusercontent.com/4025665/112339057-9d432e80-8cbf-11eb-9751-5b916ee66885.png)

while it should be: 

![Screenshot from 2021-03-24 16-39-26](https://user-images.githubusercontent.com/4025665/112338988-8ef51280-8cbf-11eb-82e6-74994fa3b937.png)

Also, before, a human placed the inputs so it to have a specific order but here it's alphabetically ordered. I prefer not to follow the alphabetical order because some inputs makes more sense in a specific order. Can you just add the new section leaving the rest untouched?

Thanks!

Reply:
Sure.